WordPress/Classes/Networking/TaxonomyServiceRemoteREST.h

```diff
@@ -2,6 +2,20 @@
 #import "TaxonomyServiceRemote.h"
 #import "SiteServiceRemoteREST.h"
 
+extern NSString * const TaxonomyRESTCategoryIdentifier;
+extern NSString * const TaxonomyRESTTagIdentifier;
+
+extern NSString * const TaxonomyRESTIDParameter;
```

Comment:
Considering that these constants are used for internal parsing in `TaxonomyServiceRemote.m` only at this time, I'd suggest removing them from the public header.

NOTE: I know some of these are being used for tests, but my recommendation normally would be not to use these constants directly in the tests, but have redundant constants in the tests.  That way, part of what the test does is make sure these values are not changed by mistake.  (An example of the good kind of redundancy).

Just an opinion though, feel free to disagree.